lib/datadog/appsec/contrib/active_record/instrumentation.rb

```diff
@@ -67,6 +67,11 @@ module ExecQueryAdapterPatch
             def exec_query(sql, *args, **rest)
               Instrumentation.detect_sql_injection(sql, adapter_name)
 
+              # Rails 5 instruduces `prepare:` keyword argument in exec_query method.
+              # Ruby 2.5 expands empty keyword arguments to empty hash,
+              # which raises ArgumentError due to incorrect number of arguments.
+              return super(sql, *args) if Gem.loaded_specs['activerecord'].version < Gem::Version.new('5.0')
```

Comment:
I think if we have exec_query/3 should not we pass all 3 arguments to the super? Or shouldn't the patch be different on its own than?